composer.json

```diff
@@ -16,11 +16,11 @@
     ],
     "require": {
         "php": ">=5.4",
-        "symfony/framework-bundle" : "~2.3",
-        "fsi/admin-bundle" : "1.1.x-dev"
+        "symfony/framework-bundle" : "^2.3",
+        "fsi/admin-bundle" : "~1.1@dev"
```

Comment:
i think we have an agreement to leave ~1.1@dev dependencies and introduce 2.0 versions and ~2.0 dependencies after admin-bundle 2.0.0 release